resources/feed-template.xml

```diff
@@ -24,10 +25,12 @@
             <guid isPermaLink="false" />
             <pubDate />
             <dc:creator />
+            <media:thumbnail url="" width="" height="" />
```

Comment:
You are missing an 's' character here :) 

http://www.rssboard.org/media-rss#media-thumbnails